fem/tmop.cpp

```diff
@@ -2106,6 +2107,7 @@ void TMOP_Integrator::EnableLimiting(const GridFunction &n0,
 {
    EnableLimiting(n0, w0, lfunc);
    lim_dist = &dist;
+   if (PA.enabled) { EnableLimitingPA(n0); }
```

Comment:
Shouldn't this be inside the other `EnableLimiting` method -- to enable PA limiting in both cases?

Reply:
Now it's called in both cases.